src/osd/OSDMap.cc

```diff
@@ -5820,6 +5820,26 @@ void OSDMap::check_health(health_check_map_t *checks) const
       d.detail.swap(nearfull_detail);
     }
   }
+
+  // POOL_NO_REDUNDANCY
+  if (g_conf()->mon_warn_on_pool_no_redundancy)
```

Comment:
@sseshasa looks like https://github.com/ceph/ceph/pull/30689 will make things easier for you

Reply:
Yes, it will. I can rebase after this goes in.

Reply:
@sseshasa Do you still plan to rebase this PR? #30689 has been merged in the meanwhile.

Reply:
@smithfarm Yes I will rebase this PR. Thanks for notifying.